docs/tutorials/transformation-and-normalization/transformations-with-sql.md

```diff
@@ -24,9 +26,9 @@ Anyway, it is possible to short-circuit this process \(no vendor lock-in\) and h
 
 This could be useful if:
 
-1. you have different usage than analytics that could be handled with these initial data in raw JSON format. 
-2. you can implement your own Transformer \(even in a different language such as Java or in Spark for example, or another transformation tool: DBT or Dataform\) 
-3. you want to customize and change how the data is normalized with your own queries.
+1. You have different usage than analytics that could be handled with these initial data in raw JSON format. 
```

Comment:
I might be understanding this one wrong, but I think this clears it up? @ChristopheDuong 
```suggestion
1. You have a use-case not related to analytics that could be handled with data in its raw JSON format. 
```